callisto/notification/api.py

```diff
@@ -23,6 +23,16 @@ class NotificationApi(AbstractNotification):
     report_filename = "report_{0}.pdf.gpg"
     from_email = '"Reports" <reports@{0}>'.format(settings.APP_URL)
 
+    @classmethod
+    def get_user_site(self, user):
+        '''Takes in a user model, and should return a site_id
+
+        example:
+            for an Account model 1 to 1 with User that has a site attribute
+            return user.account.site.id
+        '''
+        return None
```

Comment:
hey sorry I didn't notice this before, but should this return settings.SITE_ID so that matching notification doesn't fail automatically if it's not overridden? (not sure if that would work or not)